js/ui/status/payg.js

```diff
@@ -85,6 +85,10 @@ class PaygIndicator extends PanelMenu.SystemIndicator {
         if (!this._paygManager.initialized)
             return _('Getting time…');
 
+        // if PAYG is disabled, nothing should be showing this label
+        if (!this._paygManager.enabled)
```

Comment:
Looks good, and the other places we call `paygManager.timeRemainingSecs()` look safe.